modules/images/webp-uploads/settings.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Settings for the webp uploads module.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Register media settings field to generate multiple mime WebP & JPEG output.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_register_media_settings_field() {
+	// Register field setting for the processing.
+	register_setting( 'media', 'perflab_generate_webp_and_jpeg' );
+}
+add_action( 'init', 'webp_uploads_register_media_settings_field' );
+
+/**
+ * Add media settings field to generate multiple mime WebP & JPEG output.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_add_media_settings_field() {
+	// Add settings field.
+	add_settings_field(
+		'perflab_generate_webp_and_jpeg',
+		__( 'Generate WebP and JPEG', 'performance-lab' ),
+		'webp_uploads_generate_webp_jpeg_setting_callback',
+		'media',
+		'uploads',
+		array( 'class' => 'hidden' )
+	);
+}
+add_action( 'admin_init', 'webp_uploads_add_media_settings_field' );
+
+/**
+ * Callback function for the media settings perflab_generate_webp_and_jpeg field.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_generate_webp_jpeg_setting_callback() {
+	?>
+	<tr>
+		<td class="td-full">
+			<label for="perflab_generate_webp_and_jpeg">
+				<input name="perflab_generate_webp_and_jpeg" type="checkbox" id="perflab_generate_webp_and_jpeg" aria-describedby="perflab_generate_webp_and_jpeg_description" value="1"<?php checked( '1', get_option( 'perflab_generate_webp_and_jpeg' ) ); ?> />
+				<?php esc_html_e( 'Generate JPEG files in addition to WebP', 'performance-lab' ); ?>
+			</label>
+			<p class="description" id="perflab_generate_webp_and_jpeg_description"><?php esc_html_e( 'Enabling JPEG in addition to WebP can improve compatibility, but will effectively double the filesystem storage use of your images.', 'performance-lab' ); ?></p>
+		</td>
+	</tr>
```

Comment:
Having the `<tr>` and `<td>` wrappers in here breaks the HTML. It does _look_ correct, but the resulting HTML is broken (see output below):
```html
<tr class="hidden"><th scope="row">Generate WebP and JPEG</th><td>	<tr>
    <td class="td-full">
        <label for="perflab_generate_webp_and_jpeg">
            <input name="perflab_generate_webp_and_jpeg" type="checkbox" id="perflab_generate_webp_and_jpeg" aria-describedby="perflab_generate_webp_and_jpeg_description" value="1" />
            Generate JPEG files in addition to WebP			</label>
        <p class="description" id="perflab_generate_webp_and_jpeg_description">Enabling JPEG in addition to WebP can improve compatibility, but will effectively double the filesystem storage use of your images.</p>
    </td>
</tr>
</td></tr>
```

I see what you're trying to do here, but we have to make sure to get valid HTML output. It's a bit challenging because WordPress's `do_settings_fields()` doesn't really give us the flexibility we need (please take a look at the code of that function to see how we can get correct HTML). Given the limitations, I would suggest the following:
* Remove the `<tr>` and `<td>` wrappers from the code here.
* Instead, add a `</td><td class="td-full">` at the beginning. This will ensure we close the `<td>` that WordPress core opens, and then open our own, which WordPress core then closes.
* In the `add_settings_field()` call above, change the `hidden` class to some other specific class name.
* We can then add our own `<style>` tag with the rules we need - essentially hide the `th` and `td` elements within, except the `td.td-full`.

I hope that makes sense, it's a bit confusing and unfortunate that WordPress makes that so hard.

Reply:
> * Instead, add a `</td><td class="td-full">` at the beginning. This will ensure we close the `<td>` that WordPress core opens, and then open our own, which WordPress core then closes.

If we add `</td><td class="td-full">` then also it add an empty `<td></td>` so I remove it that https://github.com/WordPress/performance/pull/537/commits/ab65615023d29ee1839cae928a23e67739947d32 and add one CSS rule for full width and padding.

Reply:
Thanks @mukeshpanchal27, this looks good now.